Demo/AppDelegate.swift

```diff
@@ -14,7 +14,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
 
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
This does not work in XCode 9.4 - see 34f7beb4c4d215158bbf67eb4d1e5b78cc25e336

Reply:
yes, it was changed to swift 4.2 and Xcode 10.1

Reply:
Does the existing code not working in 10.1?

Reply:
yes, its doesn't work with new Xcode